bench/src/main/scala-2/cats/collections/bench/DietAddBench.scala

```diff
@@ -77,16 +77,16 @@ class DietAddBench extends BestCaseRangesList {
 
   @Benchmark
   def scalazDievAdd(): Unit = {
-    scalazValues.foldLeft(Diev.empty[Int])((d, r) => d + r)
+    scalazValues.foldLeft(Diev.empty[Int])((d, r) => d + r): Unit
```

Comment:
Scala complains here and below that a non-unit value gets discarded.
Ascribing the expression type to `Unit` helps to calm it down.